### DISCUSSION TITLE
PF and IPF generated only for step0 from simulation.sim

### DISCUSSION THREAD

**Discussion by Srijan7Y** [neperfepx/neper#883] [Q&A] [answered]
Hii, I'm Srijan
I am getting the ipf and pf only for step0 of my FEPX Simulation. It is not showing the maps for other steps, and even giving the error for ipf  of step 1 to 10.

Here are the commands: 

 **neper -V simulation.sim -step 4 -space pf -print image1**

![image1](https://github.com/neperfepx/neper/assets/148672326/bb8dd775-650b-4866-82c4-f269bd94d009)

 **neper -V simulation.sim -step 0 -space pf -print image0**

![image0](https://github.com/neperfepx/neper/assets/148672326/5e3cfdcd-ba0d-4de4-b6bc-70fa89f8f18b)

For ipf, it is giving error.
Here are the commands: 

srijan@LAPTOP-PJRHOE65:/mnt/d/Neper&FEPX$ **neper -V simulation.sim -step 4 -space ipf -print image2**

========================    N   e   p   e   r    =======================
Info   : A software package for polycrystal generation and meshing.
Info   : Version 4.9.0
Info   : Built with: gsl|muparser|opengjk|openmp|nlopt|libscotch (full)
Info   : Running on 8 threads.
Info   : <https://neper.info>
Info   : Copyright (C) 2003-2022, and GNU GPL'd, by Romain Quey.
Info   : No initialization file found (`/home/srijan/.neperrc').
Info   : ---------------------------------------------------------------
Info   : MODULE  -V loaded with arguments:
Info   : [ini file] (none)
Info   : [com line] simulation.sim -step 4 -space ipf -print image2
Info   : ---------------------------------------------------------------
Info   :   - Reading arguments...
Info   : Loading mesh...
Info   :     [i] Parsing file `simulation.sim/inputs/simulation.msh'...
Info   :     [i] Parsed file `simulation.sim/inputs/simulation.msh'.
Info   : Reconstructing mesh...
Info   : Printing pole figure...
Aborted

For step 0 it is giving the ipf: 
 **neper -V simulation.sim -step 0 -space ipf -print image3**

![image3](https://github.com/neperfepx/neper/assets/148672326/8658cc7c-1814-4a58-a3a9-1747e7f71114)

I have installed asymptote and texlive-full.
All tests were successfull for neper and FEPX.
 
I also went through issue #554 and tried it out, but didnt get the results. Here are the commands for the same:

srijan@LAPTOP-PJRHOE65:/mnt/d/Neper&FEPX$ **neper -V simulation.sim -step 4 -space pf -imageformat asy -print img**

========================    N   e   p   e   r    =======================
Info   : A software package for polycrystal generation and meshing.
Info   : Version 4.9.0
Info   : Built with: gsl|muparser|opengjk|openmp|nlopt|libscotch (full)
Info   : Running on 8 threads.
Info   : <https://neper.info>
Info   : Copyright (C) 2003-2022, and GNU GPL'd, by Romain Quey.
Info   : No initialization file found (`/home/srijan/.neperrc').
Info   : ---------------------------------------------------------------
Info   : MODULE  -V loaded with arguments:
Info   : [ini file] (none)
Info   : [com line] simulation.sim -step 4 -space pf -imageformat asy
         -print img
Info   : ---------------------------------------------------------------
Info   :   - Reading arguments...
Info   : Loading mesh...
Info   :     [i] Parsing file `simulation.sim/inputs/simulation.msh'...
Info   :     [i] Parsed file `simulation.sim/inputs/simulation.msh'.
Info   : Reconstructing mesh...
Info   : Printing pole figure...
Info   :     [o] Writing file `img.asy'...
Info   :   - Computing symbols...
Info   :     [o] Wrote file `img.asy'.
Info   : Printing scale...
Info   : Elapsed time: 0.151 secs.
========================================================================

 **asy -f png -render 4.428421 img.asy -o img.png**

![img](https://github.com/neperfepx/neper/assets/148672326/44e6954f-924f-46b4-bb98-3bc18b9379ba)

I did a simple compression test for Copper. I am getting the deformed shapes , as visualised by -dataeltcol ori.

**Please help. @rquey @mkasemer**

**Comment by rquey** [comment, marked as answer]
This discussion has been closed as obsolete. Please repost if necessary.
